caf-audit-service-container/pom.xml

```diff
@@ -296,16 +296,16 @@
                                             <dependencySet>
                                                 <outputDirectory>webapps/${webserviceWarName}</outputDirectory>
                                                 <includes>
-                                                    <include>com.github.cafaudit:${webserviceWarName}</include>
+                                                    <include>com.github.cafaudit:caf-audit-service</include>
                                                 </includes>
                                                 <unpack>true</unpack>
                                                 <useProjectArtifact>false</useProjectArtifact>
                                             </dependencySet>
                                             <!-- Including the swagger ui in the container image. -->
                                             <dependencySet>
-                                                <outputDirectory>webapps/${webserviceWarName}-ui</outputDirectory>
+                                                <outputDirectory>webapps/caf-audit-service-ui</outputDirectory>
```

Comment:
@dermot-hardy Why change this one back when the token is used in line #299 and the names will always match apart from the -ui on the end?